src/main/java/com/minecolonies/colony/buildings/AbstractBuilding.java

```diff
@@ -279,8 +300,7 @@ public void writeToNBT(NBTTagCompound compound)
         if (s == null)
         {
             throw new IllegalStateException(this.getClass() + " is missing a mapping! This is a bug!");
-        }
-        else
+        } else
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this "else" keyword to a new dedicated line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ARightCurlyBraceDifferentLineAsNextBlockCheck)